ocr-numbers.md

```diff
@@ -1,36 +1,74 @@
-## Step 1
+# Step One
 
-A simple binary font has been constructed using only pipes and
-underscores.
+To begin with, convert a simple binary font to a string containing 0 and 1.
```

Comment:
do you mean for step one to be a single digit, or multiple digits? If it is to be a single digit, "0 and 1" seems to imply the single digit should simultaneously be both, which would not be